### PR TITLE
Feature/communications mail endpoint

### PR DESCRIPTION
**Pull request by stefanhar**
# \<Description\>

https://app.asana.com/0/1183498341031105/1198200112715893

## What

- Endpoint for sending emails
- Supports multiple email templates

## Why

- To route messages from web to service desk

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/email-service/src/lib/email.service.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export class EmailService {
       }
     } catch (e) {
       this.logger.error(`Fatal error when sending email: ${e}`, e)
+      throw e
```
Comment (gudjong) [inline libs/email-service/src/lib/email.service.ts:74]: 👍

**Review by gudjong — APPROVED** [review]

File: libs/email-service/src/lib/email.service.ts (hunk 1 above)
Comment (pshomov) [reply libs/email-service/src/lib/email.service.ts:74]: 👍🏻

**Review by pshomov — APPROVED** [review]
LGTM

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by gudjong — APPROVED** [review]

**Pull request merged**